app/(header-default)/dirigeants/[slug]/_component/sections/entreprise/RolesInfos.tsx

```diff
@@ -0,0 +1,35 @@
+import { IRole } from '#models/rne/types';
+import React from 'react';
+import DisambiguationTooltip from './DisambiguationTooltip';
+
+export default function RolesInfos({ roles }: { roles: IRole[] }) {
+  return roles && roles.length > 1 ? (
+    <ul>
+      {roles.map((role, index) => (
+        <li key={index}>
+          <span>{role.label}</span>
+          <DisambiguationTooltip
+            dataType="rôle"
+            isInIg={role.isInIg}
+            isInInpi={role.isInInpi}
+          />
+          {index < roles.length - 1 && <br />}
+        </li>
+      ))}
+    </ul>
+  ) : (
+    <React.Fragment>
+      {roles?.map((role, index) => (
+        <React.Fragment key={index}>
+          <span>{role.label}</span>
+          <DisambiguationTooltip
+            dataType="rôle"
+            isInIg={role.isInIg}
+            isInInpi={role.isInInpi}
+          />
+          {index < roles.length - 1 && <br />}
+        </React.Fragment>
+      ))}
+    </React.Fragment>
+  );
```

Comment:
> This is nice, but I think you could drop the ternary and the `<br/>` by using `<div>` rather than fragment or `<ul>/<li>`

Last version: 
<img width="1272" alt="image" src="https://github.com/user-attachments/assets/e959b7b0-9573-44bf-9b4b-41172f6a585a">